src/platform/brand-consolidation/components/SubmitSignInForm.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import isBrandConsolidationEnabled from '../feature-flag';
+
+export default function SubmitSignInForm({ children, startSentence }) {
+  if (!isBrandConsolidationEnabled()) {
+    return <span>{children}</span>;
+  }
+
+  return (
+    <span>
+      <a href="https://www.accesstocare.va.gov/sign-in-help">
```

Comment:
`noopener`?

Reply:
I forgot this was in our new tab domains list. So normally, no, this wouldn't need `noopener` because it's not opening in a new tab, but I need to change that, so I will have to add `noopener`.